### ISSUE TITLE
Move to MavenCentral

### ISSUE THREAD

**Issue by jeggy** [aPureBase/KGraphQL#134]
JCenter is closing down and KGraphQL should move to mavenCentral instead.

**Comment by DVDAndroid** [comment]
I don't know if you noticed, but the latest version on MavenCentral of this library is literally `version`

[https://search.maven.org/artifact/com.apurebase/kgraphql/version/jar](https://search.maven.org/artifact/com.apurebase/kgraphql/version/jar)

**Comment by jeggy** [comment]
Haha, I have done something wrong then 😄 
I started working on getting this on mavenCentral yesterday. I will close this issue when I have successfully moved over there, I will also make sure to publish some older versions also.

**Comment by jeggy** [comment]
Found the issue, I was just using a string instead of the variable `version` :smile:  But I have published it now and it should be available soon on MavenCentral.

I will start publishing older versions now, so anyone can move to MavenCentral

**Comment by Ischca** [comment]
KDataLoader v0.1.1 seems to be in JCenter😕

```kotlin
Could not resolve all files for configuration ':runtimeClasspath'.
> Could not find de.nidomiro:KDataLoader:0.1.1.
  Searched in the following locations:
    - https://repo.maven.apache.org/maven2/de/nidomiro/KDataLoader/0.1.1/KDataLoader-0.1.1.pom
  If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
  Required by:
      project : > com.apurebase:kgraphql:0.17.5
```

**Comment by jeggy** [comment]
Yes that's true. For now you can fix this by adding this to your build gradle: `implementation("de.nidomiro:KDataLoader:0.1.1")`

**Comment by nidomiro** [comment]
I'm on it ;)

**Comment by nidomiro** [comment]
KDataLoader is online on Maven Central :)

https://repo1.maven.org/maven2/de/nidomiro/

**Comment by jeggy** [comment]
I have released `0.17.7` 👌

This line `implementation("de.nidomiro:KDataLoader:0.1.1")` is not required anymore.

**Comment by jeggy** [comment]
We are available on Maven Central: https://search.maven.org/search?q=g:com.apurebase%20kgraphql
